db/new_seeds/base/add_schools_and_local_authorities.rb

```diff
@@ -16,18 +16,15 @@ def add_school_to_local_authority(school:, local_authority:, lead_providers:, co
 
       scenarios = Random.rand(1..4).times.map do
         NewSeeds::Scenarios::Participants::Mentors::MentoringMultipleEctsWithSameProvider
-          .new(
-            school:,
-            lead_provider:,
-            number: Random.rand(1..4), # number of mentees
-          )
+          .new(school:, lead_provider:)
```

Comment:
Feels like this was better before as if you don't call `add_mentees` the scenario "name" doesn't make sense?

Reply:
Yes, I have made a refactor and simplified a few methods.